pkg/reconciler/v1alpha1/taskrun/entrypoint/entrypoint.go

```diff
@@ -195,14 +195,26 @@ func getRemoteImage(image string, kubeclient kubernetes.Interface, build *buildv
 		return nil, fmt.Errorf("Failed to create k8schain: %v", err)
 	}
 
-	// this will first try to authenticate using the k8schain,
-	// then fall back to the google keychain,
-	// then fall back to anonymous
-	mkc := authn.NewMultiKeychain(kc, google.Keychain)
+	// this will first try to anonymous
+	// the fall back to authenticate using the k8schain,
+	// then fall back to the google keychain
```

Comment:
Can you add a comment stating that this will (currently) fail if gcloud isn't available?